hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/annotations/ServicesV049.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
```

Comment:
nit: 
```suggestion
 * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Reply:
Here and in a few new classes

Reply:
WIll fix, but spotless did that ... let me see if it does it again (if so I'll report it).